webui/react/src/pages/ExperimentList.tsx

```diff
@@ -138,6 +138,14 @@ const ExperimentList: React.FC<Props> = ({ project }) => {
   const { settings, updateSettings, resetSettings, activeSettings } =
     useSettings<ExperimentListSettings>(settingsConfig);
 
+  const tableOffset = (() => {
```

Comment:
We really should have `total` tracked in the `InteractiveTable` component so we can do this calculation there. With this change the shown url parameters are slightly different but the functionality should be the same.